examples/rtl_433_mqtt_hass.py

```diff
@@ -111,7 +111,7 @@
 
 # Fields that get ignored when publishing to Home Assistant
 # (reduces noise to help spot missing field mappings)
-SKIP_KEYS = NAMING_KEYS + [ "time", "mic", "mod", "freq", "sequence_num",
+SKIP_KEYS = NAMING_KEYS + [ "mic", "mod", "freq", "sequence_num",
```

Comment:
I haven't personally found the time sensors to be useful. Are there types of readings where they are?

Reply:
Since I posted this, I've now seen the use. There are sensors that may send a message periodically even if they haven't changed. Having an entity for this in Home Assistant allows us to easily know when the message was last received, and not just changed value.